setup.py

```diff
@@ -9,7 +9,7 @@
     'flask>=0.10.1',
     'marshmallow>=2.0.0',
     'webargs>=0.18.0',
-    'apispec>=0.17.0',
+    'apispec==1.0.0',
```

Comment:
Hello @DStape,
Why is it pinned to `1.0.0`?
I suppose there should be `apispec>=1.0.0` instead.

Reply:
Good catch, will fix just now...